index.js

```diff
@@ -37,7 +37,16 @@ export {
     Video,
     View,
     WebView
-} from './src/components.js';
+} from './src/components/components.js';
+
+export {
+  Alignment,
+  AnchorPoint,
+  FontStyle,
+  FontWeight,
+  Orientation,
+  TextAlign
+} from './src/components/base-types/enums'
```

Comment:
Do we have different types in addition to `base-types` if not, lets rename `base-types` to just `types`.

Reply:
Not yet, but I assume that I will move all component elements to some folder (or keep them in the parent folder of `base-types`/`types`).